maat-court-data-api/src/main/java/gov/uk/courtdata/dces/request/LogContributionProcessedRequest.java

```diff
@@ -0,0 +1,17 @@
+package gov.uk.courtdata.dces.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+public class LogContributionProcessedRequest extends LogProcessedRequest {
```

Comment:
Extending was used as there is work ahead for adding an endpoint for fdcs processing to be logged as well. So this will allow for ease of adding it in.